src/app/portlets/dot-edit-page/shared/models/dot-rendered-page.model.ts

```diff
@@ -3,11 +3,48 @@ import { DotPage } from './dot-page.model';
 import { DotTemplate } from './dot-template.model';
 import { DotEditPageViewAs } from '@models/dot-edit-page-view-as/dot-edit-page-view-as.model';
 
-export interface DotPageRender {
+export interface DotPageRenderResponse {
     layout?: DotLayout;
     page: DotPage;
     containers?: any;
     template?: DotTemplate;
     canCreateTemplate: boolean;
     viewAs: DotEditPageViewAs;
+    numberContents: number;
+}
+export class DotPageRender implements DotPageRenderResponse {
+    constructor(private params: DotPageRenderResponse) {
+    }
+
+    get layout(): DotLayout {
+        return this.params.layout;
+    }
+
+    get page(): DotPage {
+        return this.params.page;
+    }
+
+    get containers(): any {
+        return this.params.containers;
+    }
+
+    get template(): DotTemplate {
+        return this.params.template;
+    }
+
+    get canCreateTemplate(): boolean {
+        return this.params.canCreateTemplate;
+    }
+
+    get viewAs(): DotEditPageViewAs {
+        return this.params.viewAs;
+    }
+
+    get numberContents(): number {
```

Comment:
This shouldn't be here, this is part of the state, this class is just for wrap the response and that we can extend the state, but this should be exactly as the response.